protobuf/v1/registration.proto

```diff
@@ -13,6 +13,14 @@ message RegisterRequest {
   string grpc_endpoint = 2;
   // uuid generated by the instance on startup
   google.protobuf.StringValue instance_uuid = 3;
+  // latest api version supported by the dataplane
+  APIVersion api_version = 4;
```

Comment:
what about adding another message which has the version for now, and in the future we can add other things like capabilities? Or is it better to keep it separate?

Reply:
Capabilites now will be a part of host, MayaStorFeatures as per v1 api.